tests/functional/provisioning/test_dev_path_mapping_file.py

```diff
@@ -792,3 +793,91 @@ def test_dev_path_mapping_heketi_node_delete(self):
+
+        # Get pvs info
+        pvs_info_before = openshift_storage_libs.get_pvs_info(
+            self.node, self.node_ip, self.devices_list)
+        self.detach_and_attach_vmdk(
+            self.vm_name, self.node_hostname, self.devices_list)
+        pvs_info_after = openshift_storage_libs.get_pvs_info(
+            self.node, self.node_ip, self.devices_list)
+
+        # Compare pvs info before and after
+        for (path, uuid, vg_name), (_path, _uuid, _vg_name) in zip(
+                pvs_info_before[:-1], pvs_info_after[1:]):
+            self.assertEqual(
+                uuid, _uuid, "pv_uuid check failed. Expected:{},"
+                "Actual: {}".format(uuid, _uuid))
+            self.assertEqual(
+                vg_name, _vg_name, "vg_name check failed. Expected:"
+                "{}, Actual:{}".format(vg_name, _vg_name))
+
+        for pvc in pvc_names:
+            openshift_ops.resize_pvc(self.node, pvc, expand_pvc_size)
+            openshift_ops.verify_pvc_size(self.node, pvc, expand_pvc_size)
+
+        for vol_name in vol_names:
+            vol_info = heketi_ops.heketi_volume_info(
+                self.h_node, self.h_server, vol_name['heketi_vol'], json=True)
+            self.assertEqual(
+                vol_info["size"], expand_pvc_size, "Failed to validate volume"
+                " size. Expected {}, Actual{}".format(
+                    expand_pvc_size, vol_info["size"]))
+
+            volume = vol_name['gluster_vol']
+            volume_info = volume_ops.get_volume_info(
+                self.gluster_servers[0], volume)
+            self.assertIsNotNone(
+                volume_info, "Failed to get volume info for volume {}".format(
+                    volume))
+            self.assertEqual(
+                volume_info[volume]['brickCount'], "6", "Failed to validate "
+                "brick count. Expected {}, Actual {}".format(
+                    "6", volume_info[volume]['brickCount']))
+
+        # Pod names list
+        pod_names = [pod_name for _, (_, pod_name) in dc_name.items()]
+        self.assertIsNotNone(
+            pod_names, "Failed to get the pod name from {}".format(dc_name))
+
+        # Creating IO basesize and count
+        base_size, count = 4096, 100000
+        async_obj, mounts = [], '/mnt'
+        cmd_run_io = 'oc rsh {} dd if=/dev/urandom of={} bs={} count={}'
+        err_msg = "Failed to wait for IO to complete"
+        for pod in pod_names:
+            for file_count in range(2):
+                _file = '{}/file{}'.format(mounts, file_count)
+                async_op = g.run_async(
+                    host=self.node, command=cmd_run_io.format(
+                        pod, _file, base_size, count))
+                async_obj.append(async_op)
+
+            for i, proc in async_obj:
+                ret, _, _ = proc.async_communicate()
+                self.assertFalse(ret, err_msg)
+
+        self._get_space_use_percent_in_app_pod(pod_names[0])
```

Comment:
plz refer to https://github.com/gluster/glusterfs-containers-tests/pull/22/files#diff-8a3cd13ec6c45a8d79a90e0ec1ee85c3707967870338b2c3532d895daa56c3acR790
I order to check for the check for space and percentage.

Reply:
done